webapp/src/app/admin/contacts/contacts.component.ts

```diff
@@ -145,16 +139,7 @@ export class ContactsComponent implements OnInit {
                         await this.uData.save();
                         const url = this.location.prepareExternalUrl("/register?ephemeral=" +
                             newUser.keyIdentity._private.toHex());
-                        let host = window.location.host;
-                        if (Defaults.Testing) {
-                            if (host.match(/local[1-9]?/)) {
```

Comment:
I would very much like to keep this functionality: when in testing mode and the host is one of `local[1-9]?`, it should take the next `local` as hostname. This is a very simple but good enough way to have multiple identities on the same browser for testing...

Reply:
Yeah, I see the usefulness of this, I put it back.